src/main/java/vg/civcraft/mc/civmodcore/legacy/materials/IMaterial.java

```diff
@@ -0,0 +1,17 @@
+package vg.civcraft.mc.civmodcore.legacy.materials;
+
+import org.bukkit.Material;
+
+public interface IMaterial {
```

Comment:
Why would we want this?

Reply:
Are you referring to the interface itself or the legacy materials section altogether?

Reply:
The legacy materials (which are also this interfaces justification). We have no intention to have our plugins support multiple versions at once and especially won't make them compatible to past versions (1.12, 1.13), so I don't see valid use cases for this. Further having this encourages patterns we don't want in our code, namely keeping/suporting configs in a legacy format as long as possible, which forces us to get more and more creative as Mojang/Spigot keep changing stuff. Given how few people use our stuff its a very unnecessary burden to put on ourselves. Additionally this is also a decently sized maintenance burden if stuff starts getting built around it, so I am opposed to it.